<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setTime</class>
 <widget class="QDialog" name="setTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Time</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="hoursLabel">
      <property name="text">
       <string>Hours</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="minutesLabel">
      <property name="text">
       <string>Minutes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="hoursInput"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="minutesInput"/>
    </item>
    <item row="4" column="1">
     <widget class="QDialogButtonBox" name="ConfirmButton">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="isBreakTime">
      <property name="text">
       <string>Break time</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
